src/apps/moldft/input

```diff
@@ -2,7 +2,7 @@ dft
   xc LDA
   maxsub 5
 end
-  #xc GGA_X_PBE .75 GGA_C_PBE 1. HF_X .25
+  #xc "GGA_X_PBE .75 EXTERNAL_PARAMETERS {kappa: 0.7, mu: 0.3} GGA_C_PBE 1. EXTERNAL_PARAMETERS {gamma: 0.05, beta: 0.08} HF_X .25"
```

Comment:
leave the example before as well.
add small comment what kappa, mu, gamma, beta are.